### PR TITLE
fix median

### PR DESCRIPTION
**Pull request by topsenw**

### CODE REVIEW

File: src/utils/QCCalculator.cpp [hunk 1]
```diff
@@ -156,9 +156,11 @@ class TOPPQCCalculator :
 
   float calculateSNmedian (MSSpectrum<Peak1D>& spec, bool norm = true)
   {
+	if (spec.size() == 0) return 0;
```
Comment (mwalzer) [inline src/utils/QCCalculator.cpp]: Looks good. Leads to the question though, why there would be any empty spectrum.

Comment (cbielow) [inline src/utils/QCCalculator.cpp]: Guess how the bug was found (real data) :-)

**Comment by cbielow** [comment]
C++ style check failed (see http://cdash.openms.de/testDetails.php?test=39516145&build=220397)
i.e. please replace tab with spaces.

**Comment by cbielow** [comment]
thanks Timo.
Will merge.

**Pull request merged**